src/OrbitGl/App.cpp

```diff
@@ -676,9 +676,18 @@ void OrbitApp::OnErrorsWithPerfEventOpenEvent(
 
 void OrbitApp::OnWarningInstrumentingWithUprobesEvent(
     orbit_grpc_protos::WarningInstrumentingWithUprobesEvent
-    /*warning_instrumenting_with_uprobes_event*/) {
-  // TODO(b/232072696): Display WarningInstrumentingWithUprobesEvent in the Capture Log.
-  ORBIT_UNREACHABLE();
+        warning_instrumenting_with_uprobes_event) {
+  main_thread_executor_->Schedule([this, warning_instrumenting_with_uprobes_event = std::move(
+                                             warning_instrumenting_with_uprobes_event)]() {
+    std::string message = "Uprobes likely failed to instrument some functions:\n";
+    for (const auto& function :
+         warning_instrumenting_with_uprobes_event.functions_that_failed_to_instrument()) {
+      absl::StrAppend(&message, "* ", function.error_message(), "\n");
+    }
```

Comment:
User space instrumentation would work in this case, right? You could, after the loop, add something like 

`absl::StrAppend(&message, "You might want to use instrumentation method \"Orbit\" in the settings dialog instead."`

Reply:
Example: https://screenshot.googleplex.com/BGppv94PCYQYkY7

Reply:
Oh, yes, getting pointed around in a circle is not that great. But I agree that in the majority of cases this is useful. I think this is fine as is now.